syncmaster/scheduler/__main__.py

```diff
@@ -0,0 +1,33 @@
+# SPDX-FileCopyrightText: 2023-2024 MTS PJSC
+# SPDX-License-Identifier: Apache-2.0
+import asyncio
+import logging
+
+from syncmaster.scheduler.transfer_fetcher import TransferFetcher
+from syncmaster.scheduler.transfer_job_manager import TransferJobManager
+from syncmaster.settings import Settings
+
+logger = logging.getLogger(__name__)
+
+
+async def main():
+    settings = Settings()
+    transfer_fetcher = TransferFetcher(settings)
+    transfer_job_manager = TransferJobManager(settings)
+    transfer_job_manager.scheduler.start()
+
+    while True:
+        logger.info("Looking at the transfer table...")
+        transfers = await transfer_fetcher.fetch_updated_jobs()
+
+        if transfers:
+            logger.info(f"Found {len(transfers)} updated transfers: {transfers}")
```

Comment:
Please use %-string syntax for logging instead of f-strings. Some instruments, like Sentry, rely on grouping log entries by message key, and f-strings produce unique message keys for each run.